File: Source/DroolTool.Models/DataTransferObjects/Announcement/AnnouncementUpsertDto.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DroolTool.Models.DataTransferObjects.Announcement
{
    public class AnnouncementUpsertDto
    {
        public int? AnnouncementID { get; set; }
        public string AnnouncementTitle { get; set; }
        public DateTime AnnouncementDate { get; set; }
        public string? AnnouncementLink { get; set; } 
    }
}

File: DroolTool.Web/src/app/shared/models/announcement/announcement-upsert-dto.ts
export class AnnouncementUpsertDto{

    public AnnouncementID?: number;
    public AnnouncementTitle:string;
    public AnnouncementLink?: string;
    public AnnouncementDate: Date;

    constructor(obj?: any){
        Object.assign(this, obj);
    }
}
